### PR TITLE
Create review layout

### PR DESCRIPTION
**Pull request by tim-macphail**
## Created the review layout as per figma designs. 

As you can see the design changes quite a bit on mobile devices. This is probably not the prettiest way to handle the responsiveness, but the point is it works, and now everyone else can start developing their components into this page.

https://user-images.githubusercontent.com/70448914/216712899-b3eceb7e-7182-4b93-b4da-345b4f7310a8.mp4

The next challenge with this design is managing app state / props for the data received from the server. Since it's all on one page it should be easier now. Another challenge will be having smooth transition animations from pages on mobile view.

### CODE REVIEW

File: client/src/pages/Review/Review.tsx [hunk 1]
```diff
@@ -1,39 +1,87 @@
...
+    <div className="flex h-screen">
+      <nav
+        className={classnames(
+          "w-full", "md:w-64", "bg-gray-600", "h-full", "fixed", "left-0", "top-0", "flex", "flex-col", "p-4", "items-center", "z-0", !courseId && "z-20"
+        )}
+      >
+        <div className="flex flex-col h-full">
+          {courses.map((course) => (
+            <Link to={`/review/${course}`} key={course}>
+              <Button variant={courseId === course ? "filled" : "tonal"}>
```
Comment (cloudyyoung) [inline client/src/pages/Review/Review.tsx:23]: `Button` supports `to` as a prop. 

suggestion```
<Button to={`/review/${course}`} key={course} variant={courseId === course ? "filled" : "tonal"}>
```

File: client/src/pages/Review/Review.tsx [hunk 2]
```diff
@@ -1,39 +1,87 @@
...
+                {format(course)}
+              </Button>
+            </Link>
+          ))}
+        </div>
       </nav>
+      <div className="flex flex-col w-full ml-0 md:ml-64 z-10">
```
Comment (cloudyyoung) [inline client/src/pages/Review/Review.tsx:30]: The main container must have a maximum width to adapt wide screen, and the container should be center-aligned

Screenshot 2023-02-03 at 7 48 37 PM

Screenshot 2023-02-03 at 7 48 45 PM

**Review by cloudyyoung — COMMENTED** [review]
The absence of layout in this PR would not block any frontend devs from implementing their assigned component, components are to assemble into the layout at the very end.

Having one comment regarding wider screens display.

**Comment by tim-macphail** [comment]
> The absence of layout in this PR would not block any frontend devs from implementing their assigned component, components are to assemble into the layout at the very end.
> 
> Having one comment regarding wider screens display.

This is mainly to help Bradley design the app top bar with some template code of how breakpoints work, trust me I was talking to him on Thursday and having this page layout in place will make it much easier for him. 

So just to clarify, the pdf viewer starts at the midway point of the screen, and fills the whole screen until it reaches max-width?

**Pull request merged**